### PR TITLE
App: Added templates

### PR DESCRIPTION
**Pull request by techmsi**
https://trello.com/c/S9t5UOZ2/1-bare-bones-instance-generator

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -8,9 +8,15 @@ module.exports = generators.Base.extend({
     generators.Base.apply(this, arguments);
   },
 
+  copying: function () {
+    this.copy('_gitignore', '.' + 'gitignore');
+    this.copy('_csscomb.json', '.' + 'csscomb.json');
+    this.copy('_eslintrc', '.' + 'eslintrc');
```
Comment (yoshokatana) [inline generators/app/index.js]: shouldn't these use the `this.templatePath()` and `this.destinationPath()` functions?

File: generators/app/templates/_gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+*.gz
+*.iml
+
+npm-debug.log
+node_modules
+coverage.html
+Dockerfile
+
+# config (get local from someone on your team if you need it)
+local.yaml
```
Comment (yoshokatana) [inline generators/app/templates/_gitignore]: this should probably be changed to `local.yml` (or maybe have both?)

Comment (techmsi) [inline generators/app/templates/_gitignore]: Both.

File: generators/app/index.js (hunk 1 above)
Comment (techmsi) [inline generators/app/index.js]: Yes.

File: README.md [hunk 3]
```diff
@@ -3,9 +3,26 @@ A Yeoman generator for Clay
...
+
+
+* `.gitignore`
+* `.csscomb.json`
+* `.eslintrc`
+
 ## Component Generator
 
-You can use the component generator to generate new components in your clay repo.
+You can use the component generator to generate new **components** in your clay repo.
```
Comment (yoshokatana) [inline README.md]: good call

File: generators/app/templates/_package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "package",
+  "version": "0.0.0",
+  "main": "app.js",
+  "scripts": {
+    "test": "node_modules/.bin/eslint app.js components sites gulp filters && node_modules/.bin/mocha",
+    "start": "node app.js"
+  },
+  "dependencies": {
+    "@nymdev/amphora": "^0.15.0"
```
Comment (yoshokatana) [inline generators/app/templates/_package.json]: oh! this should be `^1.0.0` now. I should also update it in the pr-blog repo

**Comment by yoshokatana** [comment]
:+1: though we should update the amphora dep to 1.0.0

File: README.md (hunk 3 above)
Comment (techmsi) [inline README.md]: :+1: Thanks.

File: generators/app/templates/_package.json (hunk 4 above)
Comment (techmsi) [inline generators/app/templates/_package.json]: What should the package name be?

Comment (techmsi) [inline generators/app/templates/_package.json]: Changed package name to be what user entered.

File: generators/app/index.js [hunk 5]
```diff
@@ -1,16 +1,117 @@
...
+  _ = require('lodash');
 
 module.exports = generators.Base.extend({
   constructor: function () {
     generators.Base.apply(this, arguments);
+    this.log('Welcome to the ' + chalk.yellow.bold('Clay Instance') + ' generator!');
+
+    // Store user-inputed appname
+    this.argument('appname', { type: String, required: false });
+    this.appname = this.app_name || this.appname;
```
Comment (yoshokatana) [inline generators/app/index.js]: don't need app_name

**Comment by yoshokatana** [comment]
:+1:

**Pull request merged**